source/locales.tex

```diff
@@ -1043,8 +1043,8 @@
 in case of a parse error.
 
 \pnum
-Within this clause it is unspecified whether one virtual function calls another
-virtual function.
+Within subclause \ref{locale.categories} it is unspecified whether one virtual
+function calls another virtual function.
```

Comment:
This linebreak doesn't move us towards semantic line breaks.
Either leave the original (bad) linebreak (i.e. don't touch) or make it right.

Reply:
Sorry, that was the result of a `gq` in vim on autopilot :) Fixing.